package.json

```diff
@@ -64,7 +64,6 @@
     "react-moveable": "^0.21.6",
     "react-photo-gallery": "^8.0.0",
     "react-pure-render": "^1.0.2",
-    "react-transition-group": "^4.4.1",
```

Comment:
niiice

Reply:
Haah I'm gonna end up using it for the transitions between tip pages. 

Went to go implement a custom transition group for this, then figured we didn't need the complexity or extra testing surface when we can use `react-transition-group` and it's 4kb.

Didn't end up needing it for this components animations with the setup I went with, so I figured I'd remove it from this PR and add it back in when it was actually relevant to the code I was adding in

Reply:
even better, thanks max